improver_tests/precipitation_type/snow_splitter/test_snow_splitter.py

```diff
@@ -82,14 +84,54 @@ def precip_rate_cube() -> Cube:
     return precip_cube
 
 
+@pytest.fixture()
+def precip_acc_cube() -> Cube:
+    """Set up a r, y, x cube of precipitation accumulation"""
+    data = np.full((2, 2, 2), fill_value=1, dtype=np.float32)
+    precip_cube = set_up_variable_cube(
+        data,
+        name="lwe_thickness_of_precipitation_amount",
+        units="m/s",
```

Comment:
The units for precip accumulation would typically be "m". This would also need an update in the tests.

Reply:
Done